### PR TITLE
Add tasks documentation

### PR DESCRIPTION
**Pull request by th1988**

### CODE REVIEW

File: source/includes/_tasks.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+{
+  "data": [
+     {
+       "id": "f7559f2a-8f1c-461e-8a9b-7efea5564edb",
+       "type": "tasks",
+       "attributes": {
+         "name": "Implement CompanyMood",
+         "description": "Implement continious Mood monitoring",
+         "progress": '100%',
```
Comment (SweeD) [inline source/includes/_tasks.md]: There is no DB attribute named `progress`
We have the field `percent` which is an integer.

**Review by SweeD — CHANGES_REQUESTED** [review]
Looks good to me, but one field is wrong. 
Thanks, good work! 👍

**Comment by th1988** [comment]
Hey @SweeD,
changed it 👍

**Review by SweeD — APPROVED** [review]

**Comment by SweeD** [comment]
Looks good to me. 👍 

Plese go ahead and 🚢 at your will. 🎉

**Comment by th1988** [comment]
@SweeD would you please deploy the changes? Or if you tell me how, I can do it by myself :)

**Comment by SweeD** [comment]
Just use the `.deploy`script.

**Pull request merged**